packages/api/.gitignore

```diff
@@ -152,6 +152,7 @@ dist
 .pnp.*
 
 ### VisualStudioCode ###
+.vscode
```

Comment:
It was showing up in my untracked files trying to sneak in to the repository.  Adding this removed it for me.

Reply:
Well that should be a directory of things but there are two things we're intentionally including in the repo. What file was untracked?

Reply:
'/api/.vscode/launch.json'
'/api/.vscode/settings.json'

These files were created by vscode when I was running jest for testing.

Reply:
Sounds like you opened the `api` folder in vscode. Open `Project-X.code-workspace` in vscode to avoid this, it's at the root of the project. It'll create those files at the root of the project which will then be auto ignored

Reply:
This line has been removed.